src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types';
 import {
   Button,
   Form, Icon, StatefulButton, Toast, useToggle,
```

Comment:
Split component imports into different lines.

Reply:
This component is currently like that so I won't change it